aspnetcore/razor-pages/razor-pages-conventions/samples/3.x/SampleApp/README.md

```diff
@@ -1,12 +1,12 @@
 # ASP.NET Core Model Providers Sample
 
-This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://docs.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
+This sample illustrates use of Razor Pages custom route and page model providers. This sample demonstrates the features described in the [Razor Pages route and app conventions](https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-convention-features) topic.
```

Comment:
Same as above, Adding to broken link tab. Try https://learn.microsoft.com/aspnet/core/razor-pages/razor-pages-conventions

Reply:
Pre-existing issue, keeping for now.